src/pages/posts/s02e03.md

```diff
@@ -0,0 +1,34 @@
+---
+title: DevOps Party Games - Don't Call It A Comeback (EMEA)
+subtitle: Broadcast live at <a href = "https://www.timeanddate.com/worldclock/fixedtime.html?msg=DevOps+Party+Games&iso=20210209T20&p1=3903" target = "_blank">Tuesday 9 February 20:00 UTC+1</a> 
+excerpt: >-
+  Streaming live at <a href = "https://www.timeanddate.com/worldclock/fixedtime.html?msg=DevOps+Party+Games+A+New+Challenger+Appears+%28EMEA%29&iso=20210209T20&p1=3903" target = "_blank">20:00 UTC+1</a><br> on Tuesday 9 February
```

Comment:
Same: on vs at?

Reply:
No, this one leads with the time so "at" makes sense :)